pipeline/beam_tables.py

```diff
@@ -31,6 +31,8 @@
 from pipeline.lookup_country_code import country_name_to_code
 from pipeline.metadata.flatten import Row
 from pipeline.metadata import flatten
+from pipeline.ip_metadata_values import IpMetadataValuesInterface
```

Comment:
I'm using -`Interface` because the non-interface name is already taken for the actual class implementation

ex for caida:
`CaidaIpMetadataInterface` / `CaidaIpMetadata` (actual implementation) / `FakeCaidaIpMetadata` (fake for testing)

What pattern should I change these three to?